afdko/PKGBUILD

```
@@ -3,32 +3,34 @@
 
 pkgname=afdko
 pkgver=4.0.1
-pkgrel=1
+pkgrel=2
 pkgdesc='Adobe Font Development Kit for OpenType'
 arch=(x86_64)
 url="https://github.com/adobe-type-tools/$pkgname"
-license=(custom)
+license=(Apache-2.0)
 _py_deps=(booleanoperations
           brotli # for fonttools[woff]
           defcon
           fontmath
-          fontparts
           fontpens # for defcon[pens]
           fonttools
           fs # for fonttools[ufo]
-          lxml # for fonttools[lxml] and defcon[lxml]
+          lxml # for defcon[lxml] and fonttools[lxml]
           tqdm
           ufonormalizer
           ufoprocessor
           unicodedata2 # for fonttools[unicode]
           zopfli) # for fonttools[woff]
-depends=(python
-        "${_py_deps[@]/#/python-}"
-        psautohint)
+depends=(gcc-libs
+         glibc
+         libxml2
+         python
+         "${_py_deps[@]/#/python-}")
 makedepends=(cmake
              git # Upstream Issue: https://github.com/adobe-type-tools/afdko/issues/1407
-             python-setuptools-scm
-             python-scikit-build)
+             ninja
+             python-{build,installer,wheel}
+             python-{scikit-build,setuptools-scm})
```

My preference is one per line except the PEP517 stuff which isn't specific to the project.

```suggestion
             python-scikit-build
             python-setuptools-scm)
```